src/devices/Rtc/Rtc.csproj

```diff
@@ -10,7 +10,7 @@
       <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
     </ProjectReference>
     <ProjectReference Include="..\Units\Units.csproj" />
-    <Compile Include="..\Common\Iot\Device\Common\NumberHelper.cs" />
+    <ProjectReference Include="..\Common\Iot\Device\Common\CommonHelpers.csproj" />
```

Comment:
I would keep the Compile item here, and similar on the CreditCard project, that way you won't have to change the API from internal to public.

Reply:
Done (pending commit)